include/wx/msw/private/tlwgeom.h

```diff
@@ -113,6 +113,41 @@ class wxTLWGeometry : public wxTLWGeometryBase
             return false;
         }
 
+        if (m_placement.showCmd != SW_SHOWMAXIMIZED && m_placement.showCmd != SW_SHOWMINIMIZED)
+        {
+            RECT rect;
+            ::GetWindowRect(tlw->GetHWND(), &rect);
+            // Height and width should be the same unless the user performed
+            // an Aero Snap operation.
+            if ((rect.bottom - rect.top) !=
+                (m_placement.rcNormalPosition.bottom - m_placement.rcNormalPosition.top)
+                ||
+                (rect.right - rect.left) !=
+                (m_placement.rcNormalPosition.right - m_placement.rcNormalPosition.left))
+            {
+                MONITORINFO mi;
+                mi.cbSize = sizeof(mi);
```

Comment:
A very minor improvement:
```suggestion
                WinStruct<MONITORINFO> mi;
```